.travis.yml

```diff
@@ -116,17 +129,20 @@ before_install:
   - composer config -g github-oauth.github.com ${GITHUB_TOKEN}
 
 install:
+  # make sure index works and sends us to installation page
+  - composer travis:install
   - bash .scripts/travis/check_commit_msgs.sh
+  - php -S localhost:8888 index.php &
+  - sleep 3 # give Web server some time to bind to sockets, etc
+  - curl -o - http://localhost:8888/ | grep "<title>Elgg Install&#58; Welcome</title>"
```

Comment:
@mrclay Any clue why it doesn't catch the exception? It works fine for me locally.